include/secp256k1.h

```diff
@@ -599,57 +613,85 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_ec_pubkey_negate(
     secp256k1_pubkey *pubkey
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2);
 
-/** Tweak a private key by adding tweak to it.
- * Returns: 0 if the tweak was out of range (chance of around 1 in 2^128 for
- *          uniformly random 32-byte arrays, or if the resulting private key
- *          would be invalid (only when the tweak is the complement of the
- *          private key). 1 otherwise.
+/** Tweak a secret key by adding tweak to it.
+ * Returns: 0 if the resulting secret key would be invalid (only when the tweak
+ *          is the negation of the secret key). 1 otherwise.
  * Args:    ctx:    pointer to a context object (cannot be NULL).
- * In/Out:  seckey: pointer to a 32-byte private key.
- * In:      tweak:  pointer to a 32-byte tweak.
- */
+ * In/Out:  seckey: pointer to a 32-byte secret key. The secret key should be
+ *                  valid according to secp256k1_ec_seckey_verify. If this
+ *                  function returns 0, seckey will be some unspecified
+ *                  value. (cannot be NULL)
+ * In:       tweak: pointer to a 32-byte tweak. Must be in the same range as secret
```

Comment:
"must" is similar to "should" here. I guess it's better to specify the consequences if it's not in range, then the function will return 0.

Reply:
fixed